### PR TITLE
Bug 1407623 - Add documentation for short-term projects.

### PR DESCRIPTION
**Pull request by adngdb**

### CODE REVIEW

**Comment by adngdb** [comment]
@peiying2 How does that look?

File: tools/pontoon/adding_new_short_term_project.md
Comment (flodolo) [inline tools/pontoon/adding_new_short_term_project.md]: nits:
* `...` should be either `etc.` or `…` (single Unicode character). 
* Use ’ instead of ' (apostrophe instead of straight single quote): Pontoon's -> Pontoon’s. This also later in the doc.

File: tools/pontoon/adding_new_short_term_project.md
Comment (flodolo) [inline tools/pontoon/adding_new_short_term_project.md]: Use italic, or “” (curly double quotes) to highlight text.

**Review by flodolo — COMMENTED** [review]
Can you add the link also to SUMMARY.md? Otherwise the HTML file won't be generated in GitBook.

I'll leave the final review to Pei, but the doc looks clear to me.

**Comment by adngdb** [comment]
@flodolo comments addressed. @peiying2, ping for the review. :)

**Comment by peiying2** [comment]
@adngdb , apologize for taking this long to review the doc. Finally I have a real project so I can give it an official run. The instruction is clear and I can follow through the steps and the output is what I expected.

I do have some questions for you to see if you have tips or advice on how to process the project in as little time as possible. After I tried the process on staging, all looked good. When I switch to production side, I have to front load the strings, select the languages, and save the project. Only after this step, when the project technically has gone public, that I can "manage strings" - add comments, check that the strings are all in the right place and in the correct sequence. 

What I am afraid is that as soon as the strings go public, localizers start to translate right away. Are there workarounds to either shorten my prep time or disable the project until I am ready to publish it (Save Project):

A: Can I Save Project and Disable it at the same time so I can "Manage strings"? Once done, uncheck Disabled box? 
B: Alternatively, if there is a way to upload the file I prepared at staging to production (since I have checked it there already) to reduce the time needed prepping on production? 

Thank you!

**Comment by peiying2** [comment]
For this email project, since we are covering all the email languages, I will set it up the standard way. 

Is there a way to preserve the comment in the output file? https://docs.google.com/spreadsheets/d/1Fe1LiqF5E6GDpvkMzds4ZOcyc85091K6LGCV6Bmcfr4/edit#gid=485606406. This has a lot of comments. Alternatively, I can copy the translated strings to the original file from the requester.

**Comment by adngdb** [comment]
@peiying2 You can indeed disable the project, save it, make all your changes and the de-disable it to make it available to localizers. I'm afraid at the moment there's no better solution for you. We talked about having a better form of import at a second step to this, but we wanted to make sure that the feature was useful first. Now that you're using it, how about we take some time to work on the various improvements you'd like to see? Maybe setup a meeting next week? 

For the moment, I see two features: add comments to the export document, and allow importing that same document (so that you can setup on stage, then just have to export / import to prod).

**Comment by peiying2** [comment]
@adngdb I can totally use the Disabled feature for the time being. I know you are super booked with other requests. If you have time to work on this, assuming the solution is not too complicated, then let's resolve this. Next Wednesday and Thursday mornings (my time) look good. Please find a slot to have a quick chat. Thank you!

**Comment by peiying2** [comment]
@adngdb this is the first: https://pontoon.mozilla.org/id/engagement-firefox-rocket-keyword-aso/database/. Given most of the requests come from marketing, can we have a high level project name, and sub-category (check Engagement project for reference). Otherwise, all the requests have a very flat structure, and may flood a locale dashboard very quickly. All the individual file name will be called "database". This is only a "wish". If this means spending a lot of your time to improve the feature, then don't do it. Thanks!

**Comment by peiying2** [comment]
@adngdb, I have added more strings to the existing project. Initially I added to the production side directly but on the status bar, it shows 100% completion, so I clicked on the link, at file level, it does show the strings waiting to be completed, but between the % icon and the status bar, a localizer won't know something new is added to an existing project at higher level. I went to the staging site to add these strings, it didn't fix the problem. The Sync button is for repo type of project, not database type, but I tried it too, :). It didn't work. We need to have the status feature fixed.

**Comment by flodolo** [comment]
~Can we not use a documentation PR to discuss feature issues? That belongs to a bug.~
I see [there is a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1436889).

I'd also like to understand the status of this PR.

**Comment by flodolo** [comment]
Periodic ping…

**Comment by adngdb** [comment]
I think we should be able to merge this, for even if the feature could be improved, that should happen in bugs, and in my opinion it shouldn't block merging the documentation. @peiying2 what do you think?

**Comment by peiying2** [comment]
Let's merge this.

**Review by peiying2 — APPROVED** [review]
Good to go. Issues stated will be followed up with bugs.

**Pull request merged**
